### DISCUSSION TITLE
Annotations just like Zotero

### DISCUSSION THREAD

**Discussion by GH4TP** [sumatrapdfreader/sumatrapdf#4324] [Ideas]
There are a few suggestions in discussions here regarding annotations which I think are well represented by the way Zotero annotates. Colors can be selected at the time a selection is made or set once. Annotations does not have a side panel but does have a right-click modal. Clicking once on an annotation and pressing Delete deletes an annotation. A rectangle can be drawn around a graphic to annotate it. Annotations can be left-click to add a comment or comments can be dropped by selecting a posty-type note icon from the top menu and placed anywhere. Having used both Sumatra and Zotero I think that latter has nailed the annotation workflow ideally and is worth shamlessly copying. Just a personal opinion, of course.

**Comment by GitHubRulesOK** [comment]
Inside a PDF colour has to be set before draw, so yes a colour picker would help to preset before the next shape, otherwise you need to hold annotation temporarily back to amend colour after.
- Clicking once on an annotation and pressing Delete deletes an annotation. 
   - there were issues that brushing delete accidentally removed annotations so currently EITHER rightclick or CONTROL the DELETE KEY
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/bd239d93-5381-4e10-b430-79469abf9cf8)

- A rectangle can be drawn around a graphic to annotate it.
   -  = hold CTRL & left mouse drag (Marquee in Acrobat terminology) Press A to highlight any colour either preset or changed in side menu HOWEVER highlight is not "square" which is the alternate annotation for "boxing"
